base/v0_2_exp/translate_test.go

```diff
@@ -339,7 +348,88 @@ func TestTranslateFile(t *testing.T) {
 				},
 			},
 			[]translate.Translation{},
-			"error at $.contents.local: open " + filepath.Join(filesDir, "file-2") + ": no such file or directory\n",
+			"error at $.contents.local: open " + filepath.Join(filesDir, "file-missing") + ": no such file or directory\n",
+			base.TranslateOptions{
+				FilesDir: filesDir,
+			},
+		},
+		// inline and local automatic file compression
+		{
+			File{
+				Path: "/foo",
+				Contents: Resource{
+					// should compress
+					Inline: util.StrToPtr(zzz),
+				},
+				Append: []Resource{
+					{
+						// should compress
+						Local: util.StrToPtr("file-2"),
+					},
+					{
+						// should not compress
+						Local:       util.StrToPtr("file-2"),
+						Compression: util.StrToPtr("gzip"),
```

Comment:
If I'm reading this right both this case & the one below shouldn't be compressed because they are specifying the compression. If so it might be more obvious to choose a different compression string to make it more obvious at a glance.